src/upcoming-components/Accordion/Accordion.stories.tsx

```diff
@@ -2,9 +2,9 @@ import { Story, Meta } from '@storybook/react';
 import React from 'react';
 
 import { Accordion, Props } from './Accordion';
+import Heading from '../../components/Heading';
```

Comment:
I updated all the imports that were pointing to components still in `src/components` due to type errors.